sources/packages/web/src/components/aest/institution/modals/ApproveDenyDesignationModal.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <v-form ref="approveDenyDesignation">
+    <modal-dialog-base
+      :showDialog="showDialog"
+      :title="title"
+      :max-width="730"
+      scrollable
```

Comment:
mm..but this custom custom component does not have a `scrollable`

Reply:
The modal-dialog-base already has the `scrollable` defined and the Manage Users modal was supposed to have it working. Is this modal different in some way?
![image](https://user-images.githubusercontent.com/61259237/198093297-13b838ab-6dd6-46e5-a617-8a7e9e9c4377.png)

Reply:
![image](https://user-images.githubusercontent.com/62901416/198107226-edf38d81-f1bb-46d6-8635-ce9e406d2b99.png)